.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ concurrency:
 
 jobs:
   check:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest 
```

Comment:
Does this CI also replenish the cache?
Alternatively, How does one populate the cache cache?